kitt_bar_app/current_batch.rb

```diff
@@ -57,59 +60,59 @@ def fetch_api_data
   end
 
   def user_ticket
-    @api_data.dig('tickets')&.find { |ticket| ticket.dig('is_mine') }
+    @tickets&.find { |ticket| ticket.is_mine? }
   end
 
   def parse_batch_status
   	@color 			 	= @api_data.dig('camp', 'color') == "grey" ? "gray" : @api_data.dig('camp', 'color') || 'gray'
-  	@ticket_count = @api_data.dig('tickets')&.count || -1
+  	@ticket_count = @tickets&.count || -1
   	@lunch_break  = @api_data.dig('camp', 'on_lunch_break') || false
   end
 
   def batch_open?
-    @api_data && @api_data.dig('status').nil?
+    @api_data && @api_status.nil?
   end
 
   def toggle_duty
     return unless @api_data['on_duties']
 
-    if @api_data.dig('on_duties').map { |on_duty| on_duty['id'] }.include?(@api_data.dig('current_user', 'id'))
-      puts "🟢 On duty"
-      puts "-- 🍕 take a break | #{HttpKitt.patch(@slug, "finish")}"
+    if current_user_in_on_duty?
+      @view.append_with(body: "🟢 On duty")
+      @view.append_with(body: "-- 🍕 take a break", shell: HttpKitt.patch(@slug, "finish"))
```

Comment:
Pour moi la pizza m'induit parfois en erreur et j'ai l'impression que je vais déclencher la pause dej 🤪

=> 🥱 plutôt? ou autre

Reply:
ahah ok je mettrais la pizz sur le very upcoming and very awaited "lunch break btn"